# Space Mouse Configuration for TeleopTwistStampedJoy
wrench_to_twist:
  ros__parameters:
    # Publishing rate (Hz)
    publish_rate: 10.0
    
    # Scaling factors
   # force_scale: 1.0      # Max linear velocity (m/s)
    #torque_scale: 1.0     # Max angular velocity (rad/s)
    mass: 10.0
    inertia_x: 2.0
    inertia_y: 2.0
    inertia_z: 2.0
    linear_drag_x: 50.0
    linear_drag_y: 50.0
    linear_drag_z: 50.0
    rotational_drag_x: 50.0
    rotational_drag_y: 50.0
    rotational_drag_z: 50.0
  
    
    # Frame ID for the twist messages
    frame_id: "eoat_camera_link"
    
    # Topic names
    wrench_topic: "teleop_force_threshold"
    twist_topic: "/inspection_cell/servo_node/delta_twist_cmds"  # Updated topic for inspection cell